pkg/placement/helpers/clusters.go

```diff
@@ -29,7 +32,7 @@ func NewClusterSelector(selector clusterapiv1beta1.ClusterSelector, env *cel.Env
 		return nil, err
 	}
 	// build cel selector
-	celSelector := NewCELSelector(env, selector.CelSelector.CelExpressions)
+	celSelector := NewCELSelector(env, selector.CelSelector.CelExpressions, metricsRecorder)
```

Comment:
Do we have a general metrics for predicate? we should add a TODO if we do not have one.

Reply:
yes, we have metrics for predicate: https://github.com/open-cluster-management-io/ocm/blob/main/pkg/placement/controllers/scheduling/schedule.go#L200